.travis.yml

```diff
@@ -1,52 +1,70 @@
+language: python
+
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.cache/pip
 env:
   global:
-    - CONDA_DEPS="pip flake8 pytest numpy scipy"
+    - EXTRA_WHEELS="https://5cf40426d9f06eb7461d-6fe47d9331aba7cd62fc36c7196769e4.ssl.cf2.rackcdn.com"
+    - PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
+    - EXTRA_PIP_FLAGS="--find-links=$EXTRA_WHEELS"
+    - PRE_PIP_FLAGS="--pre $EXTRA_PIP_FLAGS --find-links $PRE_WHEELS"
 
 matrix:
   include:
     - os: osx
+      language: generic
       env:
          - PYTHON_VERSION=2.7
          - MINICONDA_URL="https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh"
     - os: osx
+      language: generic
       env:
          - PYTHON_VERSION=3.5
          - MINICONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh"
     - os: linux
+      python: 2.7
+    - os: linux
+      python: 3.5
```

Comment:
It's probably time to add 3.6 and/or 3.7. Should probably still keep 3.5 for another year or so.